src/ethereum_optimized/fork.py

```diff
@@ -25,10 +25,10 @@
     import ethash
 except ImportError as e:
     # Add a message, but keep it an ImportError.
-    raise e from Exception(
+    raise Exception(
         "Install with `pip install 'ethereum[optimized]'` to enable this "
         "package"
-    )
+    ) from e
```

Comment:
Now this doesn't match the comment. The weird `raise e from Exception(...)` pattern was to keep the thrown exception an `ImportError`, while adding a message. With your change, it is no longer an `ImportError`.

Reply:
I've reverted the `raise .. from` commit.